explanation/intro-to/multipath.md

```diff
@@ -1,58 +1,57 @@
 
-## Storage array overview
+## Configuring multipath for a storage system
 
-It is a good idea to consult your storage vendor's installation guide for the recommended multipath configuration variables for your storage model. The default configuration will probably work but will likely need adjustments based on your storage setup.
+Before you decide to use multipath, check your storage vendor's installation guide for the multipath configuration variables that are recommended for your storage model. The default multipath configuration will probably work but will likely need adjustments based on your storage setup.
 
 ## Multipath components
 
 | Component | Description |
-| - | - |
 | `dm_multipath` kernel module | Reroutes I/O and supports **failover** for paths and path groups. |
-| `multipath command` | Lists and configures multipath devices. Normally started up with `/etc/rc.sysinit`, it can also be started up by a `udev` program whenever a block device is added, or it can be run by the `initramfs` file system. |
+| `multipath command` | Lists and configures multipath devices. While this command is normally started up with `/etc/rc.sysinit`, it can also be started up by a `udev` program whenever a block device is added, or it can be run by the `initramfs` file system. |
 | `multipathd` daemon | Monitors paths; as paths fail and come back, it may initiate path group switches. Provides for interactive changes to multipath devices. This daemon must be restarted for any changes to the `/etc/multipath.conf` file to take effect. |
 | `kpartx` command | Creates device-mapper devices for the partitions on a device. It is necessary to use this command for DOS-based partitions with multipath. The `kpartx` is provided in its own package, but the `multipath-tools` package depends on it.  |
 
 ## Multipath setup overview
 
-Setting up multipath is often a simple procedure, since it includes compiled-in default settings that are suitable for common multipath configurations. The basic procedure for configuring your system with multipath is as follows:
+The multipath setup process is usually simple because it has compiled-in default settings that are suitable for common multipath configurations. The basic procedure for configuring your system with multipath is:
 
-1.  Install the `multipath-tools` and `multipath-tools-boot` packages
+1.  Install the `multipath-tools` and `multipath-tools-boot` packages.
 
-1.  Create an empty config file called `/etc/multipath.conf`
+2.  Create an empty config file called `/etc/multipath.conf`.
 
-1.  Edit the `multipath.conf` file to modify default values and save the updated file
+3.  Edit the `multipath.conf` file to modify default values and save the updated file.
 
-1.  Start the multipath daemon
+4.  Start the multipath daemon.
 
-1.  Update initial RAM disk
+5.  Update initial RAM disk.
```

Comment:
Not a blocker, but just FYI - in markdown you don't need to number the list sequentially. Sphinx is smart enough to handle the numbering even if all the list entries are numbered with `1`, which is very convenient if you're moving things around.